yarn.lock

```diff
@@ -12857,7 +12857,7 @@ url-parse-lax@^1.0.0:
   dependencies:
     prepend-http "^1.0.1"
 
-url-parse@^1.4.3:
+url-parse@^1.4.3, url-parse@^1.4.7:
```

Comment:
Oh, we already had this dependency, interesting...

Reply:
Isn't yarn lock supposed to be automatically generated?

Reply:
Yes, yes, I wanted to say that we were already depending on _url-parse_ (in an indirect way), it was only a comment, nothing change.